libr/cons/dietline.c

```diff
@@ -325,6 +338,24 @@ static int r_line_hist_down() {
 	if (I.hist_down) {
 		return I.hist_down (I.user);
 	}
+    if (I.offset_prompt) {
+        RCore *core = I.user;
+        RIOUndo *undo = &core->io->undo;
+        if (I.offset_index >= undo->redos) {
+            return false;
+        }
+        I.offset_index++;
+        if (I.offset_index == undo->redos) {
+            I.buffer.data[0] = '\0';
+            I.buffer.index = I.buffer.length = 0;
+            return false;
+        }
+        char line[R_LINE_BUFSIZE - 1];
+        sprintf(line, "0x%"PFMT64x, undo->seek[undo->idx + I.offset_index]);
```

Comment:
use snprintf or r_str_newf and avoid abusing the stack with this char line[R_LINE_BUFSIZE] because to fill a PFMT64x you dont need more than 64 bytes. not 4096